src/ucp/api/ucp.h

```diff
@@ -1878,6 +1981,74 @@ ucs_status_ptr_t ucp_tag_send_sync_nb(ucp_ep_h ep, const void *buffer, size_t co
 
 /**
  * @ingroup UCP_COMM
+ * @brief Non-blocking stream receive operation.
+ *
+ * This routine receives a data that is described by the local address @a
+ * buffer, size @a count, and @a datatype object on the endpoint @a ep. The
+ * routine is a non-blocking and therefore returns immediately. The receive
+ * operation is considered completed when the message is delivered to the @a
+ * buffer. In order to notify the application about completion of the receive
+ * operation the UCP library will invoke the call-back @a cb when the received
+ * data is in the receive buffer and ready for application access. If the
+ * receive operation cannot be stated the routine returns an error.
+ *
+ * @note This routine cannot return UCS_OK. It always returns a request
+ *       handle or an error.
+ *
+ * @param [in]     ep       UCP endpoint that is used for the receive operation.
+ * @param [in]     buffer   Pointer to the buffer to receive the data to.
+ * @param [in/out] count    Number of elements to receive into @a buffer as
```

Comment:
@yosefe it looks like `count` should be just `[in]` parameter because actual value will be passed to `ucp_stream_recv_callback_t` and known only on completion, right?